Detectors/TOF/calibration/src/TOFFEElightReader.cxx

```diff
@@ -22,30 +22,53 @@ void TOFFEElightReader::loadFEElightConfig(const char* fileName)
   char* expandedFileName = gSystem->ExpandPathName(fileName);
   std::ifstream is;
   is.open(expandedFileName, std::ios::binary);
-  is.read(reinterpret_cast<char*>(&mFEElightConfig), sizeof(mFEElightConfig));
+  mFileLoadBuff.reset(new char[sizeof(o2::tof::TOFFEElightConfig)]);
+  is.read(mFileLoadBuff.get(), sizeof(o2::tof::TOFFEElightConfig));
   is.close();
+  mFEElightConfig = reinterpret_cast<const TOFFEElightConfig*>(mFileLoadBuff.get());
+}
+
+//_______________________________________________________________
+void TOFFEElightReader::loadFEElightConfig(gsl::span<const char> configBuf)
+{
+  // load FEElight config from buffer
+
+  if (configBuf.size() != sizeof(o2::tof::TOFFEElightConfig)) {
+    LOG(INFO) << "Incoming message with TOFFEE configuration does not match expected size: " << configBuf.size() << " received, " << sizeof(*mFEElightConfig) << " expected";
```

Comment:
Here the check triggers just a `LOG(INFO)` since the number of TRMs defined in TOF::Geo seems to be wrong, and this check currently fails. I am confirming with @noferini , then I will change it accordingly.